### ISSUE TITLE
Post-Sleep PTT(?)/Can't Talk Issue

### ISSUE THREAD

**Issue by mumble-voip** [mumble-voip/mumble#999]
Post-Sleep PTT(?)/Can't Talk Issue

Windows 7 64-bit. Mumble 1.2.4 (but this happened once with 1.2.3a, described in the verbose section below). On-board sound card, RealTek something-or-other.

Short version:
This series of steps always has the same result: 
-- Re/Start computer, start Mumble.
-- I can talk in Mumble.
-- Close Mumble, sleep computer
-- Wake computer, start Mumble, 
-- I cannot talk in Mumble.

Things that don't let me talk: 
-- closing and restarting Mumble after the sleep. 
-- rebinding/reassigning the PTT key. 

The only thing I've found that lets me talk in Mumble after this happens: 
-- restarting my computer.

Super-verbose wall-of-text version:
On Sunday June 2 2013, I brought my computer out of sleep mode and started up my Mumble client. My Mumble 1.2.3a client advised me to upgrade to 1.2.4. At this time, for the first time, I was unable to talk in Mumble after the computer had been asleep (I would push the key -- it's bound to the ~ key -- and my my mouth-icon on Mumble didn't light up on my screen and nobody could hear anything I said). I assumed it was related to needing to update, so I installed 1.2.4. 
The problem has persisted. I've googled and found some threads that sound similar to this problem, but my problem differs; in some of the other threads, the posters say that if they go into the options and re-set their PTT binding, the problem is fixed, but when I attempt to reset my binding, my problem is not fixed. The only thing that allows me to talk in Mumble again is a full shut down and restart of my computer. 

On June 18th, I created a registry key as outlined in 
http://mumble.sourceforge.net/1.2.4_Known_Issues
"Global Shortcuts stop working randomly on Windows"
"HKEY_CURRENT_USER\Control Panel\Desktop\LowLevelHooksTimeout (it does not exist by default)
This key should be of type DWORD, and its value is specified in milliseconds."
I set the value to "3e8" hexidecimal, as stated. I restarted my computer, Mumble worked fine. I exited Mumble, slept the computer, woke the computer, started Mumble, and I cannot talk.

None of the ~50+ other people that I know that use Mumble have said they have this problem. When I was notified to upgrade to 1.2.4, I made a post that they should also upgrade, so I assume at least 15-30% of them have done so already, but I have not seen anyone mention anything about having a similar issue.

Bug reports of some similarity, yet my problem is distinct from these for various reasons:

http://sourceforge.net/p/mumble/bugs/879/
http://sourceforge.net/p/mumble/bugs/636/ - this person says their problem arises when they sleep the computer with Mumble open. I have always disconnected and closed Mumble before sleeping the computer, as I seem to find that Mumble-open-while-sleeping results in robot-voice after un-sleeping. 
http://sourceforge.net/p/mumble/bugs/298/ - the original post is from 2009, so I don't know if it still counts. Both the original poster and the additional commenter on page 2 describe almost exactly the same problem that I have, except they both state that they leave Mumble open and connected while they put their computer to Sleep-mode, and that quitting Mumble and reconnecting or attempting to rebind the PTT key will fix their problem.
https://sourceforge.net/p/mumble/bugs/997/ - unlike this poster, my Mumble works perfectly fine for the entire time I want to use it, but only after a re/start of my computer.

Please let me know if I can do anything to fix the problem. Thank you for reading.

_This ticket has been migrated from sourceforge. It is thus missing some details like original creator etc.
The original is at https://sourceforge.net/p/mumble/bugs/999/ ._

*The following attachments were added on the original item:
- https://sourceforge.net/p/mumble/bugs/999/attachment/DxDiag.txt*

**Comment by ZeroAbility** [comment]
Duplicate of https://github.com/mumble-voip/mumble/issues/996
